src/app/dashboard/dashboard/dashboard.component.html

```diff
@@ -100,30 +80,18 @@
       <app-person-pool-statistics></app-person-pool-statistics>
     }
     <!-- person preview cards -->
-    <div class="scrollable-content flex-grow flex-row position-relative">
-      @if (teamService.personsWithoutTeam.length === 0) {
-        <div class="position-absolute position-absolute-full-size pointer-events-none flex-row overflow-hidden">
-          <div class="flex-grow empty-orphan-team-hint-box flex-row flex-center flex-justify-center">
-            <div>Drag persons here</div>
-          </div>
+    @for (students of [studentsWithoutTeam]; track students) {
```

Comment:
Why do we need. this for loop?

Reply:
If you move a preview card into the person pool, it will be duplicated there. The dragular element still has the old project preference, with this array we reload the entire container and the dragular element is no longer there.